yarn.lock

```diff
@@ -0,0 +1,4093 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
This project uses npm, not yarn, for lockfiles

Reply:
🤦‍♂️ d'oh. I was wondering why the diff on `yarn.lock` was so big...😅